datatracker/migrations/0003_document_intended_std_level_document_shepherd_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 5.1 on 2024-08-14 17:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('datatracker', '0002_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='document',
+            name='intended_std_level',
+            field=models.CharField(blank=True, max_length=32),
+        ),
+        migrations.AddField(
+            model_name='document',
+            name='shepherd',
+            field=models.EmailField(blank=True, max_length=254),
+        ),
```

Comment:
@ajeanmahoney - does RPC use the shepherd field? Would it change in the RPC app? Would it be sufficient to go look it up in the datatracker if needed instead of replicating it directly into the RPC app.

I ask because this is a place where the datatracker models need to change, and this is starting to leak the current design flaw from the datatracer into this app, and I'd rather not do that.

Reply:
For this pr, I am happy not to add the field as I think we're likely to rework the document model anyway.